### ISSUE TITLE
Global resource suppressions are handled differently by analyzers

### ISSUE THREAD

**Issue by hvanbakel** [dotnet/roslyn#4259]
Given an existing project with suppressions made in global suppressions like this:
```
[assembly: SuppressMessage("MyCategory", "CS1400:AvoidUnusedResource",
    MessageId = "MissingBoundingBox",
    Scope = "resource",
    Target = "Long.Namespace.Implementation.Properties.Resources.resources",
    Justification = "temporary")]
```
This suppressed the message in FxCop, but it does not in the analyzers. In order to do so, I had to change the suppression to:
```
[assembly: SuppressMessage("MyCategory", "CS1400:AvoidUnusedResource",
    MessageId = "MissingBoundingBox",
    Scope = "member",
    Target = "Long.Namespace.Implementation.Properties.Resources.#MissingBoundingBox",
    Justification = "temporary")]
```

With this adjustment the analyzer will not throw a diagnostic so this might give false positives on the analyzers.

**Comment by hvanbakel** [comment]
I just realized it might also be related to my analyzer reporting the diagnostic on a different location. In that case, on what location should I report a resource issue?

**Comment by srivatsn** [comment]
We don't seem to support scope=resource in the suppressmessageattribute. Using this workitem to track that feature. We will need to get to this before we can declare parity with FxCop.

**Comment by jinujoseph** [comment]
reported at [link](https://developercommunity.visualstudio.com/content/problem/244056/warning-suppressiong-in-global-suppression-file-no.html)
